docs/cosmology/index.rst

```diff
@@ -210,16 +210,11 @@ instantiation by passing the keyword argument ``Ob0``::
 In this case the dark matter-only density at redshift 0 is available as class
 attribute ``Odm0`` and the redshift evolution of dark and baryonic matter
 densities can be computed using the methods ``Odm`` and ``Ob``, respectively.
-If ``Ob0`` is not specified at class instantiation, it defaults to ``None`` and
```

Comment:
Should state that if `Ob0` is not specified, it will be set to `0` and thus ignored in further calculations. Only then does the next example make sense (though I'd add `cosmo.Ob(1)` as well, to show it stays zero).

Reply:
✅